config.go

```diff
@@ -117,13 +86,13 @@ func (c Config) useTLS() bool {
 	return false
 }
 
-func (c Config) getGraylogPort() uint {
+func (e *EnvConfig) getGraylogPort() uint {
 	var portString string
-	if c.getGraylogHandlerType() == graylog.UDP {
+	if e.getGraylogHandlerType() == graylog.UDP {
```

Comment:
Do you default the port to an appropriate value somewhere?

Reply:
There is no default port set. It depends on what Graylog handler type we're using but it will parse `GRAYLOG_UDP_PORT` or `GRAYLOG_TLS_PORT` respectively and panic if none is set when `GRAYLOG_HOST` is set.

Reply:
We should have a default port set because we also have a default handler type (tls) that is set. At least that is the assumption in muselog.

Reply:
@ian-axelrod ok I set a default port and tls is the default handler type